packages/terra-spacer/package.json

```diff
@@ -37,12 +37,12 @@
   },
   "scripts": {
     "compile": "npm run compile:clean && npm run compile:build",
-    "compile:clean": "$(cd ..; npm bin)/rimraf lib",
-    "compile:build": "$(cd ..; npm bin)/babel src --out-dir lib --copy-files",
+    "compile:clean": "rimraf lib",
+    "compile:build": "babel src --out-dir lib --copy-files",
     "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "$(cd ..; npm bin)/eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
-    "lint:scss": "$(cd ..; npm bin)/stylelint src/**/*.scss",
-    "props-table": "props-table ./src/Spacer.jsx --out-dir ./docs/props-table",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "props-table": "props-table ./src/List.jsx ./src/SelectableList.jsx ./src/SingleSelectList.jsx ./src/MultiSelectList.jsx --out-dir ./docs/props-table",
```

Comment:
Oh, looks like you copied & pasted what I copied and pasted. Update to out the `./src/Space.jsx`

Reply:
Oops, sorry my bad. Updated https://github.com/cerner/terra-core/pull/1011/commits/c01471dc8c818d906d45f9111b29f7063824c66e#diff-f3bfa5e0bf12fb4af2128aaf883cc625R42